weave/trace_server/migrations/001_init.up.sql

```diff
@@ -156,50 +156,52 @@ CREATE TABLE object_versions (
     refs Array(String),
```

Comment:
IGNORE: will pull out of PR, this is just what I had to do to get my dev setup working